pkg/platform/platform.go

```diff
@@ -78,13 +79,14 @@ type GetPolicyDataResult struct {
 
 // Report is a comment/note on an issue or change request.
 type Report struct {
-	ID   int64
+	ID   any
```

Comment:
Changed because GitLab notes have type `int` and GitHub comments have type `int64`.